crates/astria-sequencer-utils/src/config.rs

```diff
@@ -0,0 +1,33 @@
+use clap::Parser;
+use serde::{
+    Deserialize,
+    Serialize,
+};
+
+#[derive(Debug, Deserialize, Parser, Serialize)]
```

Comment:
Also no need for `Deserialize, Serialize` here. But a doc comment on this struct would be nice. `clap` takes that and creates a cli help msg from it.

Reply:
removed these